example.env

```diff
@@ -0,0 +1,21 @@
+# use `source .env` to load items preceded by `export` into your working environment
+
+#### Dependant & Provider Settings
+
+# To get a new token, see https://docs.celestia.org/tutorials/node-tutorial#auth-token
+# Locally running nodes can use:
+# celestia <node-type> auth admin --p2p.network <network>
+export CELESTIA_NODE_AUTH_TOKEN="never-gonna-give-you-up"
+export CELESTIA_NODE_WS="ws://localhost:26658"
+
+#### Service Settings
+
+export EQ_DB_PATH="/tmp/db-eq-service-testing"
+# Expects a socket with ip & port specified (not transport)
+export EQ_SOCKET="127.0.0.1:50051"
+export EQ_PROTO_DIR="./service/proto" 
+
+# For using the SP1 Prover network
+# More info & request getting on the white list:
+# https://docs.succinct.xyz/docs/generating-proofs/prover-network
+NETWORK_PRIVATE_KEY="succinct-luvs-u" 
```

Comment:
maybe call it SUCCINCT_NETWORK_PRIVATE_KEY in case we someday add more prover networks

Reply:
https://docs.succinct.xyz/docs/generating-proofs/prover-network/usage#sending-a-proof-request
it's set by them